src/api/store/team.py

```diff
@@ -292,17 +293,13 @@ def update_team(self, context, args) -> Tuple[dict, MassEnergizeAPIError]:
       else:  
           if parent_id == 0:
             team.parent = None
-
-      if logo:
-          # if existing logo, the string length is around 300 characters
-          # If a new logo updated, this will be the length of the file, much larger than that       
-          new_logo = len(logo) > 1000
-          if new_logo:
-            logo = Media.objects.create(file=logo, name=f"{slugify(team.name)}-TeamLogo")
-            logo.save()
-            team.logo = logo
-          else: 
-            team.logo = None 
+     
+      if logo: #now, images will always come as an array of ids, or "reset" string 
+        if logo[0] == "reset": #if image is reset, delete the existing image
```

Comment:
Would be great if you could turn the `'reset'` thing into some variable

Reply:
+ 1

Define a constant up there and reference it here